flink-clients/src/main/java/org/apache/flink/client/program/ContextEnvironment.java

```diff
@@ -161,6 +171,8 @@ public ContextEnvironmentFactory(Client client, List<URL> jarFilesToAttach,
 			this.userCodeClassLoader = userCodeClassLoader;
 			this.defaultParallelism = defaultParallelism;
 			this.wait = wait;
+			_plan = null;
+			_graph = null;
```

Comment:
I tend to use `_field` appended names instead of `this.field`. But yeah. `last*` makes more sense here.